go/service/chat_local.go

```diff
@@ -511,6 +518,13 @@ func (h *chatLocalHandler) localizeConversation(
 		return chat1.ConversationLocal{}, err
 	}
 
+	// TODO need this more validation to make sure it's not a trick?
+	conversationLocal.Info.Usernames = utils.ReorderParticipants(
+		h.udc,
+		uimap,
+		conversationLocal.Info.TlfName,
+		conversationRemote.Metadata.ActiveList)
```

Comment:
If there is no active list or equivalently it is empty, then the `ReorderParticipants` just splits the tlfname and uses the order of users from tlfname.